src/Microsoft.Framework.Runtime/NuGet/NETPortable/NetPortableProfile.cs

```diff
@@ -42,27 +42,27 @@ public NetPortableProfile(string frameworkDirectory, string name, IEnumerable<Fr
         {
             if (String.IsNullOrEmpty(frameworkDirectory))
             {
-                throw new ArgumentNullException( "version");
```

Comment:
I'll append a commit after this one to fix this. @hishamco , you should run `build verify` locally and make sure the build succeeds before submitting PRs :smile: